public/app/plugins/datasource/loki/querybuilder/operationUtils.ts

```diff
@@ -255,8 +255,11 @@ export function addNestedQueryHandler(def: QueryBuilderOperationDef, query: Loki
   };
 }
 
-export function getLineFilterRenderer(operation: string) {
+export function getLineFilterRenderer(operation: string, caseInsensitive?: boolean) {
```

Comment:
One more request. Could you please add coverage for this function in `operationUtils.test.ts`. I'd super appreciate that 🙏 

The objective in general is to always back these kind of changes with its corresponding test.

Reply:
@matyax ofc, i'll add them now!